main.nf

```diff
@@ -2033,7 +2036,7 @@ process pmdtools {
     def platypus = params.pmdtools_platypus ? '--platypus' : ''
     """
     #Run Filtering step 
-    samtools calmd -b ${bam} ${fasta} | samtools view -h - | pmdtools --threshold ${params.pmdtools_threshold} ${treatment} ${snpcap} --header | samtools view -@ ${task.cpus} -Sb - > "${libraryid}".pmd.bam
+    samtools calmd -b ${bam} ${fasta} | samtools view -h - | pmdtools --threshold ${params.pmdtools_threshold} ${treatment} ${snpcap} --header | samtools view -@ ${task.cpus - 1} -Sb - > "${libraryid}".pmd.bam
```

Comment:
`samtools calmd -b ${bam} ${fasta} | samtools view -h - | pmdtools ...` also looks like another unnecessary use of `samtools view`. 

Also there are at least three tools in the pipe chain, but the `samtools view -@ ${task.cpus - 1}` at the end is using only one core less.